tools/config-gen/config.tpl

```diff
@@ -12,8 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// GENERATED CODE - DO NOT EDIT MANUALLY.
```

Comment:
Was this removed mistakenly?

Reply:
Updated to mention the file name (to avoid confusion with config.tpl file)

Reply:
makes sense.. reverted the changes!